package.json

```diff
@@ -21,6 +21,7 @@
     "snapshot": "zx ./scripts/snapshot.mjs",
     "pretest": "run-s build snapshot",
     "test": "zx ./scripts/test.mjs",
+    "test:unit": "vitest run -u",
```

Comment:
I think you can just use `vitest` here.
`-u` is to update snapshot, and we don't want that by default.
`run` is not necessary, because `vitest` will use `watch` when developping, and `run` on CI by default.

Reply:
Resolved